### DISCUSSION TITLE
Only not animated emotes as personal emotes allowed?

### DISCUSSION THREAD

**Discussion by trieye** [night/betterttv#6881] [Q&A] [answered]
hi,
I'm unsure if this is only a translation error:
As BetterTV Pro feature it says (translated from German): "25 personal, static emotes".
Does that mean only not animated emotes are allowed as personal emotes?
When I still had Pro i could add also animated Emotes to the personal set; however it never worked, others could never see these.
Thanks in advance

**Comment by night** [comment, marked as answer]
The copy you've read may be outdated. BetterTTV Pro's personal emotes can be static or animated, but it's worth calling out that they will appear static until hover unless users have enabled them to automatically animate in the BetterTTV settings.
